documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,47 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.<br />
+To consume the ApiKeyValueMapUpload step, proceed as follows:
+
+* Get the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions for service API Management, API portal under API plan.
+* Store your service key created for SAP API Management in the the Jenkins server as a secret text.
+* Create a new Jenkins pipeline designated for the ApiKeyValueMapUpload step.
+* Execute the pipeline and validate step exection results as explained in the blog [Integration Suite Piper commands](https://blogs.sap.com/2022/01/05/orking-with-integration-suite-piper-commands/)
+The ApiKeyValueMapUpload step allows you to:
+
+* You can create new API key value map in the API portal.
+* Prevent command execution in case the/an API key value map already exists.
+* If API key value map already exist, then delete it and execute the piper step again, which create new API Key value Map.
+* ApiKeyValueMapUpload only support create operation, but not delete, get, update, which are supported in different piper steps.
```

Comment:
```suggestion
* ApiKeyValueMapUpload only supports create operation, but not delete, get, update, which are supported in different piper steps.
```
If the other piper steps exist, please reference them here.

Reply:
those commands not yet implemented, this line was added as per Roland's suggestion.

Reply:
This line seems to be misunderstanding. I wouldn't expect a step named  "Upload" to _get_ or _delete_ something. Above, you already described that you can _create_ but not _overwrite_ a key. From my point of view there is no need to mention _get_ or _delete_.

Reply:
fixed